public/js/controllers/charts_controller.js

```diff
@@ -87,12 +87,15 @@ function nightModeOptions (nightModeOn) {
 }
 
 function ticketsFunc (gData) {
-  return map(gData.time, (n, i) => {
-    return [
-      new Date(n),
-      gData.valuef[i]
-    ]
-  })
+  var data = []
+  var lastIdx = gData.time.length - 1
+  for (let i = 0; i < lastIdx; i++) {
+    data.push([new Date(gData.time[i]), gData.valuef[i]])
+    data.push([new Date(gData.time[i + 1]), gData.valuef[i]])
```

Comment:
It's somewhat unfortunate that this doubles the data length.  I was hoping the graph lib would have a plot option like "stairs" or something.

Reply:
Actually it does. I didn't see it before, but dygraphs has a `stepPlot` options. I'll fix it now.

Reply:
Excellent!  Thanks.